lumen/util.py

```diff
@@ -236,3 +243,49 @@ def extract_refs(spec, ref_type=None):
         return refs
     filtered = [ref for ref in refs if ref[1:].startswith(ref_type)]
     return filtered
+
+
+def catch_and_notify(message=None):
+    """Catch exception and notify user
+
+    A decorator which catches all the exception of a function.
+    When an error occurs a panel notification will be send to the
+    dashboard with the message and logged the error and which method
+    it arrived from.
+
+    Parameters
+    ----------
+    message : str | None
+        The notification message, by default None.
+        None will give this "Error: {e}" where e is the
+        exception message.
+
+    """
+    pn.extension(notifications=True)
```

Comment:
Can you see if this is sufficient?

```suggestion
    pn.config.notifications = True
```